dev/com.ibm.ws.transport.http/src/io/openliberty/http/netty/channel/LoggingRecvByteBufAllocator.java

```diff
@@ -34,38 +34,62 @@ public class LoggingRecvByteBufAllocator implements RecvByteBufAllocator{
     private final RecvByteBufAllocator delegate;
 
     /**
-     * Constructor that wraps an existing allocator
+     * Constructor that wraps an existing allocator to add logging in {@code allocate(...)}
+     * 
      * @param delegate
      */
     public LoggingRecvByteBufAllocator(RecvByteBufAllocator delegate){
         this.delegate = delegate;
     }
 
+    /**
+     * Returns a handle that intercepts the buffer allocation call to log read operations.
+     *
+     * @return a new {@link Handle}
+     */
     @Override
+    @SuppressWarnings("deprecation")
     public Handle newHandle() {
         return new LoggingHandle(delegate.newHandle());
     }
 
+    /**
+     * A handle that {@link #allocate(ByteBufAllocator)} call in order to log read operations.
+     * Delegates all other operations to the wrapped {@link ExtendedHandle}.
+     * 
+     * Netty has not provided a non-depracated alternative to some calls. In the
+     * meantime, the deprecation warnings are supressed.
+     */
+    @SuppressWarnings("deprecation")
     public class LoggingHandle implements Handle{
         private final Handle delegateHandle;
         private ChannelHandlerContext context;
 
-        LoggingHandle(Handle delegateHandle){
-            this.delegateHandle = delegateHandle;
+        @SuppressWarnings("deprecation")
+        LoggingHandle(Handle handle){
+            this.delegateHandle = handle;
         }
 
+        /**
+         * Sets the context for logging addresses. This is expected
+         * to be called from the pipeline to ensure the handle can see addresses.
+         *
+         * @param context the channel handler context
+         */
         public void setChannelHandlerContext(ChannelHandlerContext context){
             this.context = context;
         }
 
+        /**
+         * Logs the read request, then delegates allocation.
+         *
+         * @param allocator the {@link ByteBufAllocator}
```

Comment:
Is `allocator` what you want to use here?

Reply:
The param in the javadoc doesn't look to match what the method signature contains `public void reset(ChannelConfig config){`

Reply:
This is actually for the allocate(ByteBufAllocator) on line 90, the git diff looks a bit weird, but it should be correct.